src/index.html

```diff
@@ -53,7 +53,13 @@
         <script src="scripts/index.ts" type="module"></script>
     </head>
     <body class="dark-mode">
-        <header class="header"></header>
+        <header class="header">
+            <h1 class="">FairShare</h1>
+            <div class="btn-controller">
+                <button id="toggle-btn">Light Mode</button>
+                <button id="expense-btn">Add Expense</button>
```

Comment:
Can you please add this attribute to this button, so it will open form that I've created.
```suggestion
                <button id="expense-btn" popovertarget="form-container">Add Expense</button>
```

Reply:
add https://github.com/dhruvdhaduk-simform/fair-share/pull/5/commits/941145860e5c70f42b7e8e96c347212e1d6ffee2